src/reynir_correct/config/GreynirCorrect.conf

```diff
@@ -4572,6 +4578,7 @@ $include Storasnid_ritm.csv
 "TTN4TN": "18", "Stafsetningarvilla", "Rita ætti 'tn' í stað 'ttn' í '{original}', svo: '{correct}'."
 "U4Y": "", "Stafsetningarvilla", "Rita ætti 'y' í '{original}' þó að framburður sé oft 'u', svo: '{correct}'."
 "U4Ú": "", "Stafsetningarvilla", "Rita ætti 'ú' í stað 'u' í '{original}', svo: '{correct}'."
+"URE": "", "Orðavalsvilla", "Orðið '{original}' telst úrelt."
```

Comment:
"telst" -> "er talið"?

Reply:
Found better phrasing in the Ritmyndir documentation, changed